opentelemetry-otlp/tests/integration_test/tests/integration_tests.rs

```diff
@@ -54,6 +54,7 @@ impl TestSuite {
 async fn integration_tests() {
     trace_integration_tests().await;
     logs_integration_tests().await;
+    metrics_integration_tests().await;
```

Comment:
Suggestion for future follow-up PR: While this works fine for now, using a shared collector setup across signals could improve efficiency and maintainability. Currently, the collector is started and stopped separately for each signal.

Reply:
That’s a good point - I’ll change it while I’m here, I don’t think it should be a problem as they are collecting separate signals anyhow.